### DISCUSSION TITLE
[Suggestion]: Give it a more visible name

### DISCUSSION THREAD

**Discussion by bobbintb** [arabcoders/watchstate#275] [General] [answered]
I am loving this so far. I started using Plex but there were a few issues so I decided to try Emby. It was better in some ways, but Jellyfin looks promising so I'm kind of bouncing back and forth. Plus, my family still uses Plex so I was looking for a way to sync statuses so they can try the others without issues of sync status. I was going to make my own until I found this.

Anyway, my small suggestion is to give it a less generic name to make it more visible. It took a bit of digging to find this app and I suspect the name might contribute to that. Keep up the good work and thank you!

**Comment by arabcoders** [comment, marked as answer]
Hello,

I am glad the tool works for you. About the renaming, Sadly i think it's past the point where i can change the name as there are so many automation going on changing the name right now might break those things. However, I am very open for someone well versed in English language to help fix README/FAQ to be more clear. 

Thank you for your suggestion.
